### PR TITLE
Add lfvm support for PUSH0 

### PR DESCRIPTION
**Pull request by LuisPH3**
Part of #465

### CODE REVIEW

**Review by simonlechner — DISMISSED** [review]
LGTM

**Review by facuMH — APPROVED** [review]

File: go/vm/lfvm/opcode.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ type OpCode uint16
 const (
 	// Stack operations
 	POP OpCode = iota
+	PUSH0
+
```
Comment (HerbertJordan) [inline go/vm/lfvm/opcode.go:27]: nit: no extra new line

File: go/vm/lfvm/opcode.go [hunk 2]
```diff
@@ -226,6 +228,7 @@ const (
 	NUM_OPCODES
 )
 
+// TODO: document order of this map initialization if any.
```
Comment (HerbertJordan) [inline go/vm/lfvm/opcode.go:231]: There should be none. Since maps in Go are randomly ordered, the initialization order has no effect.

**Review by HerbertJordan — DISMISSED** [review]
Look good, thanks.

**Review by facuMH — APPROVED** [review]

**Review by HerbertJordan — APPROVED** [review]

**Review by simonlechner — APPROVED** [review]

**Pull request merged**